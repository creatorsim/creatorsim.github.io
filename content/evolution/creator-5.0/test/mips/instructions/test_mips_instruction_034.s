
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:

    li $t0, 10
    li $t1, 13
    li $t2, 45
    li $t3, -2

    mul $t4, $t0, $t1
    mul $t5, $t1, $t2
    mul $t6, $t2, $t3
    mul $t7, $t3, $t4
