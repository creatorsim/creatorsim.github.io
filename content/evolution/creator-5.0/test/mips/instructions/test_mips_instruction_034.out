
cr[PC]:0x28; ir[1,at]:0xfffffffe; ir[8,t0]:0xa; ir[9,t1]:0xd; ir[10,t2]:0x2d; ir[11,t3]:0xfffffffe; ir[12,t4]:0x82; ir[13,t5]:0x249; ir[14,t6]:0xffffffa6; ir[15,t7]:0xfffffefc; keyboard[0x0]:''; display[0x0]:''; 

